e2e/components/Tabs/Tabs-test.e2e.js

```diff
@@ -9,6 +9,7 @@
 
 const { test } = require('@playwright/test');
```

Comment:
```suggestion
const { expect, test } = require('@playwright/test');
```

Looks like this is missing and causing VRT to fail.